src/main/java/oz/yamyam_map/module/region/service/RegionSerivce.java

```diff
@@ -0,0 +1,51 @@
+package oz.yamyam_map.module.region.service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import lombok.RequiredArgsConstructor;
+import oz.yamyam_map.module.region.dto.response.RegionResponse;
+import oz.yamyam_map.module.region.entity.Region;
+import oz.yamyam_map.module.region.repository.RegionRepository;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class RegionSerivce {
+
+	private final RegionRepository regionRepository;
+
+	// 시/도와 시/도별 시군구 목록 가져오기
+	public RegionResponse getRegionResponse() {
```

Comment:
💭
어떤 dto를 응답으로 줄지는 표현영역 계층의 구현 세부사항이라 도메인 영역의 표현을 써도 좋을 것 같습니다~!

getRegionResponse 

```suggestion
	public RegionResponse getRegions() {	
```